docs/rules/no-nested-ternary.md

```diff
@@ -1,16 +1,16 @@
-# Disallow Nested Ternaries (no-nested-ternary)
+# disallow nested ternary expressions (no-nested-ternary)
 
-Nesting ternary expressions makes code unclear. The `no-nested-ternary` rule disallows the use of nested ternary expressions.
+Nesting ternary expressions can makes code more difficult to understand.
 
 ```js
 var foo = bar ? baz : qux === quxx ? bing : bam;
 ```
 
 ## Rule Details
 
-The `no-nested-ternary` rule aims to increase the clarity and readability of code by disallowing the use of nested ternary expressions.
+The `no-nested-ternary` rule disallows nested ternary expressions to increase code clarity and readability.
```

Comment:
Let’s also delete the phrase that repeats the introduction: “ to increase code clarity and readability”